recipe/spyder-menu.json

```
@@ -19,62 +19,12 @@
                     "app_user_model_id": "spyder.Spyder",
                     "command": ["{{ PREFIX }}/Scripts/spyder.exe", "%*"],
                     "file_extensions": [
-                        ".bat",
-                        ".c",
-                        ".cc",
-                        ".cfg",
-                        ".cl",
-                        ".cmd",
-                        ".cpp",
-                        ".css",
-                        ".cxx",
-                        ".desktop",
-                        ".diff",
                         ".enaml",
-                        ".f",
-                        ".f03",
-                        ".f08",
-                        ".f2k",
-                        ".f77",
-                        ".f90",
-                        ".f95",
-                        ".for",
-                        ".h",
-                        ".hh",
-                        ".hpp",
-                        ".htm",
-                        ".html",
-                        ".hxx",
-                        ".inf",
-                        ".ini",
                         ".ipy",
-                        ".ipynb",
-                        ".jl",
-                        ".js",
-                        ".json",
-                        ".m",
-                        ".md",
-                        ".nsh",
-                        ".nsi",
-                        ".patch",
-                        ".po",
-                        ".pot",
-                        ".pro",
-                        ".properties",
-                        ".pxd",
-                        ".pxi",
                         ".py",
+                        ".pyi",
                         ".pyw",
                         ".pyx",
```

```suggestion
                        ".pyx"
```
Final list item with "," results in `JSONDecodeError`